data/core/units/monsters/Skeletal_Dragon.cfg

```diff
@@ -5,21 +5,26 @@
     #not 'race=monster', because we need the not_living attribute
     race=undead
     image="units/monsters/skeletal-dragon/skeletal-dragon.png"
-    hitpoints=86
-    movement_type=undeadfly
-    movement=5
-    experience=200
+    hitpoints=75
+    movement_type=undeadfoot
+    movement=4
+    experience=220
```

Comment:
why 220? this goes against the XP conventions for lvl4 units. Should be 200

and why restrict movement to 4?